test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -2438,5 +2440,54 @@ def setup_rnn_tests():
                                            embed_params=True, opset_version=10))
 
 
+class TestQuantizedOps(unittest.TestCase):
+    def generic_test(self, model, sample_inputs):
+        model.qconfig = torch.quantization.default_qconfig
+        q_model = torch.quantization.prepare(model, inplace=False)
+        q_model = torch.quantization.convert(q_model, inplace=False)
+        torch.onnx.export(q_model, sample_inputs, os.path.join("model.onnx"), operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK)
+
+    def generic_unary_test(self, op):
+        class QModule(torch.nn.Module):
+            def __init__(self, op):
+                super(QModule, self).__init__()
+                self.quant1 = torch.quantization.QuantStub()
+                self.op = op
+
+            def forward(self, x):
+                return self.op(self.quant1(x))
+
+        self.generic_test(QModule(op), (torch.tensor([[3.0, 4.0]])))
+
+    def test_quantized_add(self):
+        class QAddModule(torch.nn.Module):
+            def __init__(self):
+                super(QAddModule, self).__init__()
+                self.quant1 = torch.quantization.QuantStub()
+                self.quant2 = torch.quantization.QuantStub()
+
+            def forward(self, x, y):
+                return torch.ops.quantized.add(self.quant1(x), self.quant2(y), 1.0, 0)
+
+        self.generic_test(QAddModule(), (torch.tensor(3.0), torch.tensor(4.0)))
+
+    def test_quantized_relu(self):
+        self.generic_unary_test(torch.nn.ReLU())
+
+    def test_quantized_linear(self):
+        class QLinear(torch.nn.Module):
+            @staticmethod
+            def forward(x, y):
+                qx = torch.quantize_per_tensor(x, 1.0, 0, torch.quint8)
+                qlinear = nnq.Linear(2, 2)
+                weight = torch.quantize_per_tensor(y, 1.0, 0, torch.qint8)
+                bias = torch.ones(2).to(torch.float)
+                out = qlinear(qx)
+                return out.dequantize()
+        x = torch.ones(2, 2, 2, 2)
```

Comment:
Can we do some real computation, instead only do one by one? this may help us cover some bugs.

Reply:
Right now we aren't converting to caffe2. This is just checking to make sure the onnx conversion flow doesn't throw error. We can revisit this test once we add onnx->c2 path for these ops as well.